src/components/Summary/InterviewSummaryModal/index.js

```diff
@@ -164,25 +76,49 @@ const InterviewSummaryModal = props => (
         let comments = [];
         let summaries = [];
         let records = [];
+        let questionid = '';
         if (data && !loading && !error) {
           const interviewResult = toInterviewResult(test);
           interviewers = interviewResult.interviewers;
           questions = interviewResult.questions;
           comments = interviewResult.comments;
           summaries = interviewResult.summaries;
           records = data.getTest.records.items;
+          questionid = data.getTest.records.items[0].id;
+          comments.sort((a, b) => a.time.localeCompare(b.time));
         }
-        let comment_count = 1;
         let new_score = '';
-        if (comments.length === 0) {
-          try {
-            new_score = AddNewScoreForm(data.getTest.records.items[0].id);
-          } catch (e) {
-            new_score = '';
-          }
-        } else {
-          new_score = '';
-        }
+        const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
Please move `timezone` and `columns` out of `InterviewSummaryModal` component since they are static constants and should not be re-created every time when rendering.